src/lib/storage/encoding_type.hpp

```diff
@@ -23,12 +23,23 @@ namespace opossum {
 
 namespace hana = boost::hana;
 
-enum class EncodingType : uint8_t { Unencoded, Dictionary, RunLength, FixedStringDictionary, FrameOfReference, LZ4 };
+enum class EncodingType : uint8_t {
+  Unencoded,
+  Dictionary,
+  RunLength,
```

Comment:
Feel free to ignore that one, but .... what about ordering them alphabetically?

Reply:
We ordered them alphabetically but got unexpected errors in the `BinaryWriterTests`, so we kept it as it is now.